include/seqan3/core/simd/detail/simd_algorithm_avx512.hpp

```diff
@@ -0,0 +1,152 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \brief Provides specific algorithm implementations for AVX512 instruction set.
+ * \author Rene Rahn <rene.rahn AT fu-berlin.de>
+ */
+
+#pragma once
+
+#include <seqan3/core/simd/concept.hpp>
+#include <seqan3/core/simd/detail/builtin_simd_intrinsics.hpp>
+#include <seqan3/core/simd/simd_traits.hpp>
+
+//-----------------------------------------------------------------------------
+// forward declare avx512 simd algorithms that use avx512 intrinsics
+//-----------------------------------------------------------------------------
+
+namespace seqan3::detail
+{
+/*!\copydoc seqan3::simd::load
+ * \attention This is the implementation for AVX512 intrinsics.
```

Comment:
```suggestion
 * \remark This is the implementation for AVX512 intrinsics.
```
and all other places in all other files as well, pleae

Reply:
I forgot what we said here, but this should be `\attention`.